CameraTutorial/Game.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Graphics.OpenGL4;
+using OpenTK.Input;
+using LearnOpenGL_TK.Common;
+
+namespace LearnOpenGL_TK
+{
+    //We now have a rotating rectangle but how can we make the rectangle move based on user input
```

Comment:
I think we should phrase this less as moving the rectangle, and more like moving the view. That's technically what it's doing behind the scenes, but we should explain that a little bit.